README.md

```diff
@@ -1,2 +1,13 @@
 # sandbox-auth
 Authentication Provider Service for Developer Sandbox
+
+## Building
+
+This project requires Podman to build. Refer to the [Podman](https://podman.io/getting-started) getting started guide for installation instructions.
+
+Execute the `build.sh` script to build the Sandbox-Auth container. 
```

Comment:
Can we have a make target for that to be consistent with the rest of our components?

Reply:
I actually wanted to steer clear of using make, as it now seems to be quite common practice to build using buildah instead in order to achieve a reproducible result no matter where the project is built.  With the build script essentially being a one line command, do we really want to require `make` on top of that?

Reply:
Would you have example of projects that use buildah?
Also, we'll probably have other tasks to run (`lint`, `test`, `test-with-coverage`, etc.). Running them with `make` is pretty helpful.

Reply:
Here's a few articles, one is from medium.com and the other two are from our Red Hat colleagues:

[https://two-oes.medium.com/building-your-first-container-application-in-go-1941919defd7](https://two-oes.medium.com/building-your-first-container-application-in-go-1941919defd7)
[https://www.redhat.com/sysadmin/building-buildah](https://www.redhat.com/sysadmin/building-buildah)
[https://developers.redhat.com/articles/go-container#let_s_get_some_code](https://developers.redhat.com/articles/go-container#let_s_get_some_code)

We can continue discussions about using make, and I'm definitely open to it if it provides us specific benefits.  It's a topic that's not really related to the intent of this PR though, so let's move the discussion to Slack or something.

Reply:
We can keep using buildah and even `.sh` scripts behind the scenes but let's use make as our interface for all project related actions. It at least will be consistent with other repos. And I like make as a unified tool in general. Let's have the same targets as we have for other repos even if behind the scenes we use different tools.